src/adr/0069-crowdsec-IP-filtering.md

```diff
@@ -0,0 +1,39 @@
+# 69. Crowdsec
+
+Date: 2022-10-26
+
+## Status
+
+Accepted (lazy consensus).
+
+Implemented. 
+
+## Context
```

Comment:
Why is fail2ban not enougth? (answers: cvery complex set up in a distributed architecture)